transport/rif_comms/client.py

```diff
@@ -62,13 +61,7 @@ def is_subscribed_to(self, rsk_address: Address) -> bool:
         :param rsk_address: RSK address which corresponds to the topic which is being checked for subscription
         :return: boolean value indicating whether the client is subscribed or not
         """
-        topic_id = self._get_peer_id(rsk_address)
-        return self.stub.HasSubscriber(
-            Subscriber(
-                peerId=topic_id,
-                channel=Channel(channelId=topic_id)
-            )
-        ).value
+        return self.stub.IsSubscribedToRskAddress(RskAddress(address=to_checksum_address(rsk_address)))
```

Comment:
is_subscribed_to returns a bool, therefore this must be 

`         return self.stub.IsSubscribedToRskAddress(RskAddress(address=to_checksum_address(rsk_address))).value`

Reply:
correct

Reply:
fixed